docs/sources/flow/reference/components/otelcol.receiver.awsfirehose.md

```diff
@@ -0,0 +1,166 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.receiver.awsfirehose/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.receiver.awsfirehose/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.receiver.awsfirehose/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/otelcol.receiver.awsfirehose/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.receiver.awsfirehose/
+description: Learn about otelcol.receiver.awsfirehose
+title: otelcol.receiver.awsfirehose
+---
+
+# otelcol.receiver.awsfirehose
+
+`otelcol.receiver.awsfirehose` recieve metrics from [Cloudwatch Metrics Streams](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html) using AWS Kinesis Data Firehose and forwards it to other `otelcol.*` components.
+
+The output format of the Metrics Stream needs to be set to JSON and the reciever needs to be accessible on the 443 port by AWS, this can be done using a Load Balancer
+
+> **NOTE**: `otelcol.receiver.awsfirehose` is a wrapper over the upstream
+> OpenTelemetry Collector `awsfirehose` receiver. Bug reports or feature requests
+> will be redirected to the upstream repository, if necessary.
```

Comment:
```suggestion
{{% admonition type="note" %}}
`otelcol.receiver.awsfirehose` is a wrapper over the upstream OpenTelemetry Collector `awsfirehose` receiver. If necessary, bug reports or feature requests are redirected to the upstream repository.
{{% /admonition %}}
```

Is it necessary to mention that bug reports are sent upstream?  I think we could remove this.